server/settings/publish_plugins.py

```diff
@@ -310,14 +289,6 @@ class BlenderSubmitDeadlineModel(BaseSettingsModel):
     enabled: bool = SettingsField(True)
     optional: bool = SettingsField(title="Optional")
     active: bool = SettingsField(title="Active")
```

Comment:
> The question is if these jobs should be actually optional at all as if they would be disabled, submit publish job will fail.

I guess the idea would have been that it maybe would allow to disable just the farm publishing support for an instance - but there creators definitely are not configured that way. So I agree - may make no sense to be optional, or even be disabled, currently?